documentation/README_development.md

```diff
@@ -57,23 +57,19 @@ To verify the current release version, consult this URL https://cdn.digitru.st/p
 test page Prebid.js/integrationExamples/gpt/digitrust_Full.html and type `DigiTrust.version` in the debug console.
 
 1.  Update `version` in `package.json`
-2.  Update `digitrustHostPath`, `digitrustRedirect`, and `digitrustIframe` in the `prod` section of `src/config/general.json`
-    to match the new version number in the URL path.
-3.  Update `digitrustHostPath`, `digitrustRedirect`, and `digitrustIframe` to the most recent prior
-    release version in the `build` section of `src/config/general.json`
-4.  Build with command `yarn build`
-5.  Commit and push your changes to master branch
-6.  git checkout candidate
-7.  git pull
-8.  git merge --no-ff master
-9.  git tag candidate-1.5.37 (update to your version number as set in step 1)
-10. git push --follow-tags  (and then wait to confirm CircleCI build is successful)
-11. git push --force origin candidate-1.5.37:release (then wait for successful CircleCI build)
-12. git checkout master
-13. git tag v1.5.37
-14. git push origin v1.5.37
-15. Go to project site on Github and draft a new release
-16. Reflect release notes in the `Release Notes` wiki page at https://github.com/digi-trust/dt-cdn/wiki/Release-Notes
+2.  Build with command `yarn build`
+3.  Commit and push your changes to master branch
+4.  git checkout candidate
+5.  git pull
+6.  git merge --no-ff master
+7.  git tag candidate-1.5.37 (update to your version number as set in step 1)
```

Comment:
is this meant to be different from the `v1.5.37` convention? I know the documentation always had this `candidate-1.5.37` convention, but I don't think it was every used.

Reply:
We could eliminate the "candidate-1.5.xx" tag. It is unnecessary. I left it in initially since I didn't have history on why it was used. We could jump straight to using "v-1.5.xx" tag convention everywhere.

Reply:
suggest we use only the `v1.X.X` convention and remove references to `candidate-X.X.X`. I think this makes steps 11 and 12 below unnecessary.

Reply:
Modified documents to eliminate the "candidate" tag. Steps 11 and 12 still needed to push tag on master and get the Release tab to work correctly. Tag is master is a direct reflection of tag in candidate branch.